ocrd_utils/ocrd_utils/logging.py

```diff
@@ -97,6 +97,7 @@ def initLogging():
         level=logging.INFO,
         format='%(asctime)s.%(msecs)03d %(levelname)s %(name)s - %(message)s',
         datefmt='%H:%M:%S')
+    logging.getLogger('').setLevel(logging.INFO)
```

Comment:
This must have slipped through earlier – I guess this was the culprit for CI failures all along, right?

Reply:
Yep, that's it. I though it'd be sufficient to set the `level` parameter, but obviously it's not the case. We live, we learn.